tests/test_plugin.py

```diff
@@ -230,3 +231,71 @@ def test_pytest_unconfigure(mocked_config):
     mocked_config.pluginmanager.unregister = mock.Mock()
     pytest_unconfigure(mocked_config)
     assert not hasattr(mocked_config, '_reporter')
+
+
+def test_pytest_addoption_adds_correct_ini_file_arguments():
+    """Test the correct list of options are available in the .ini file."""
+    expected_argument_names = [
```

Comment:
May be converted to a tuple. It consumes less memory.

Reply:
Done ✨ Thanks!